source/NPC.cpp

```diff
@@ -417,6 +495,11 @@ NPC NPC::Instantiate(map<string, string> &subs, const System *origin, const Syst
 	result.mustEvade = mustEvade;
 	result.mustAccompany = mustAccompany;
 	
+	result.passedSpawnConditions = passedSpawnConditions;
+	result.passedDespawnConditions = passedDespawnConditions;
```

Comment:
I don't have a lot of experience with this part of the ES-codebase, but I would expect those two variables to only be relevant for Instantiated NPCs, not for the template NPCs. Is it useful to copy the (default) values from the template when those variables also get initialized with the default values?

Reply:
These lines seemed necessary for this to work as I initially didn't have them and NPCs would always spawn and never despawn, ignoring their conditions.

Reply:
Through testing of removing these, they are necessary.

Reply:
The variables are also initialized as false in the template, so I don't expect any issues when just all variables including those are copied from the template.